pages/docs/styled-system/utility-hooks/use-dimensions.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: useDimensions
+package: '@chakra-ui/hooks'
+description: 'React hook to measure dimensions of the referenced element'
+---
+
+`useDimensions` is a custom hook that measures dimensions of the referenced
+element based on its box-model.
+
+## Import
+
+```js
+import { useDimensions } from '@chakra-ui/react'
+```
+
+## Return value
+
+This hook returns an object with the properties `marginBox`, `paddingBox`,
+`borderBox`, `conentBox`, `border`, `padding`, and `margin`.
+
+Each of these properties contains a nested object which provides values
+respective to that property:
+
+| Inside the `*Box` properties | Inside `border`, `padding`, and `margin` properties |
+| ---------------------------- | --------------------------------------------------- |
+| `top`                        | `top`                                               |
+| `right`                      | `right`                                             |
+| `bottom`                     | `bottom`                                            |
+| `left`                       | `left`                                              |
+| `width`                      |                                                     |
+| `height`                     |                                                     |
+| `x`                          |                                                     |
+| `y`                          |                                                     |
+| `center` (x,y)               |                                                     |
```

Comment:
Try flipping the table. I think the results will look better that way. Don't use columns for each property, just separate them with commas. Example:

```
Value:        | Contents:
--------------------------------------------------------------------------
marginBox    | top, right, bottom, left, width, height, x, y, center (x, y)
borderBox    | top, right, bottom, left, width, height, x, y, center (x, y)
paddingBox   | top, right, bottom, left, width, height, x, y, center (x, y)
contentBox   | top, right, bottom, left, width, height, x, y, center (x, y)
border       | top, right, bottom, left
padding      | top, right, bottom, left
margin       | top, right, bottom, left
```

This way you're clearly showing all of the `*Box` values and their contents.